style.css

```diff
@@ -220,3 +233,53 @@ body {
     background-color: var(--red-bg);
     color: var(--red);
 }
+
+/* CARD */
+
+.card-container {
+    display: flex;
+}
+
+.card {
+    border-radius: var(--primary-radius);
+    border: 2px solid var(--primary-bgcolor);
+    height: 25rem;
+    width:15rem;  
+    display: flex;
+    flex-direction: column;
+    margin: 0 1rem;
+    position: relative;
+}
+
+.card-wishlist {
+    position: absolute;
+    right: 0.5rem;
+    top: 0.5rem;
+}
+
+.card-img {
+    height: 15rem;
+    width: 15rem;
+    border-radius: var(--primary-radius); 
+}
+
+.card-content {
+    margin: 0rem 0.5rem 0.5rem 0.5rem;
+    display: flex;
+    flex-direction: column;
+}
+
+.card-content button {
```

Comment:
Hey rohit, the add to cart button background can be of primary and secondary color as it represents primary action.
Also the current background color mixes with the background of card.